### ISSUE TITLE
support additional architectures with autoconf

### ISSUE THREAD

**Issue by bernstei** [andreww/fox#68]
It would be nice if FoX's autoconf stuff was updated to support newer architectures, e.g. aarch64 and ppc64le.  I don't really know much about autoconf, but would hope that it's as simple as updating autoconf itself, rerunning whatever generates files like `config.sub`, and pushing those back in.  Is there anyone who already knows how to do this?

**Comment by bernstei** [comment]
I guess maybe #4 is the reason this hasn't been done before :(

**Comment by andreww** [comment]
Yes - #4 (and my lack of time and fading ability with m4) has been the barrier to this kind of thing. 

I think the exchange of the loss of some backward compatibility for a more updatable build system is fair. So yes @bernstei, a patch to update to a modern autoconf would be awesome. I need to merge a bunch of other patches and push out a new version, but I don't think this interacts with with any of that.

**Comment by andreww** [comment]
Can this be closed now or is additional work/testing needed?

**Comment by bernstei** [comment]
On Feb 23, 2021, at 12:10 PM, Andrew Walker <notifications@github.com<mailto:notifications@github.com>> wrote:

Can this be closed now or is additional work/testing needed?

If you don't mind waiting until the downstream things to update and see if that problem really is fixed, I hope it won't take long, and I'll follow up here once we know.

**Comment by bernstei** [comment]
Looks like we may also need newer versions of config.sub and/or config.guess.  Should I start a new PR for that?

**Comment by andreww** [comment]
Yes please. I wonder if it's just a case of using new ones from the autoconf source or if it will need more thought?

**Comment by andreww** [comment]
I think this is now fixed on master.

**Comment by bernstei** [comment]
On Mar 4, 2021, at 6:29 AM, Andrew Walker <notifications@github.com<mailto:notifications@github.com>> wrote:



I think this is now fixed on master.

Thanks.  I've propagated the changes, and we should have CI results on various architectures in a few days.

Noam
